app/models/story.rb

```diff
@@ -19,10 +19,16 @@ class Story
   index(updated_at: 1)
   index(aasm_state: 1)
 
+  field :age_confirm, type: Boolean, default: false
+  field :guardian_consent, type: Boolean, default: false
+
   accepts_nested_attributes_for :ore_aggregation
 
   validates_associated :ore_aggregation
 
+  validates :age_confirm, acceptance: { accept: [true, 1], message: I18n.t('global.forms.validation-errors.user-age') }, unless: :guardian_consent
```

Comment:
Could we have a validation sanity check here preventing both age_confirm and guardian_consent being accepted on one story? i.e. if a contributor is over 16, then they can not also be under 16.